src/System.Device.Gpio/System/Device/Gpio/Drivers/SysFsDriver.cs

```diff
@@ -93,6 +115,79 @@ internal TimeSpan StatusUpdateSleepTime
         }
     }
 
+    /// <summary>
+    /// Returns the list of available chips.
+    /// This can be used to determine the correct gpio chip for constructor calls to <see cref="LibGpiodDriver"/>
+    /// </summary>
+    /// <returns>A list of chips detected</returns>
+    public static IList<GpioChipInfo> GetAvailableChips()
```

Comment:
Can you mark this as Experimental? Do we want 2 entries even though there is 1 chip?

Reply:
As seen in the issue https://github.com/dotnet/iot/issues/2366 `gpiodetect` does the same and also (sometimes) reports the same chip twice. So I mimiked the behavior of the LibGpiod driver here.